### PR TITLE
DRY-up interval change code; avoid side effect of updating post_modifed_gmt

### PR DESCRIPTION
**Pull request by barryhughes**
This change has us using an update query instead of `wp_update_post()` when we modify the import interval, because the latter function has a side effect of updating `post_modified_gmt` (which we use to track the last import operation - and so is something we don't want to alter when the interval changes).

[#66838](https://central.tri.be/issues/66838)

### CODE REVIEW

**Comment by barryhughes** [comment]
Per @bordoni: will update and change to use `Tribe__Events__Aggregator__Record__Abstract::dont_change_post_modified()`, though it may make sense to relocate that to `tribe-common` at this point.

**Comment by barryhughes** [comment]
Ready for another look. Per updates to [#66838](https://central.tri.be/issues/66838) this unwinds a number of the original changes (so we no longer have activation/reactivate row actions, etc).

File: src/Tribe/Aggregator/Cron.php [hunk 1]
```diff
@@ -87,9 +87,9 @@ public function get_frequency( $search = array() ) {
 		 */
 		$found = $schedules = apply_filters( 'tribe_aggregator_record_frequency', array(
 			(object) array(
-				'id'       => 'inactive',
+				'id'       => 'on_demand',
 				'interval' => false,
-				'text'     => esc_html_x( 'Inactive (manual/on demand)', 'aggregator schedule frequency', 'the-events-calendar' ),
+				'text'     => esc_html_x( 'On demand', 'aggregator schedule frequency', 'the-events-calendar' ),
```
Comment (borkweb) [inline src/Tribe/Aggregator/Cron.php]: I think this should be `On Demand` rather than the sentence case of `On demand` as per the Central ticket.

**Comment by borkweb** [comment]
After that one casing change, this is 👍

**Pull request merged**